python/ray/workflow/tests/test_recovery.py

```diff
@@ -295,6 +296,8 @@ def foo(x):
 
 
 def test_recovery_cluster_failure_resume_all(tmp_path, shutdown_only):
+    ray.shutdown()
```

Comment:
why do we need to add this? Shouldn't it has been shutdown?

Reply:
Do you want to add `--forked` in pytest and give it a try?

Reply:
unfortunately after these fixes I cannot reproduce the case, let me try it when I encountered the same situation again